pipeline/auto-push.cloudbuild.yml

```diff
@@ -12,8 +20,8 @@ steps:
   - scripts/gen-configs.sh
 - name: google/cloud-sdk:slim
   args:
-    - scripts/upload-tests.sh
-    - gs://us-central1-ml-automation-s-bc954647-bucket/dags
+  - scripts/upload-tests.sh
+  - gs://us-central1-ml-automation-s-bc954647-bucket/dags
   entrypoint: bash
 options:
   machineType: E2_HIGHCPU_32
```

Comment:
Wondering the reason we request E2_HIGHCPU_32 (32 vcpus, 32GB)? Do you know the default machine type?

Reply:
Will discuss offline. Going to resolve this conversation first to merge the PR.